main_test.go

```diff
@@ -2,18 +2,13 @@ package main
 
 import (
 	"encoding/json"
-	"fmt"
 	"testing"
 )
 
 func TestUnmarshallSearchResponse(t *testing.T) {
```

Comment:
What is your test scenario?

Give it a look to https://github.com/golang/go/wiki/TableDrivenTests to give an idea of how tests should be performed.

Reply:
Hey Rollo I apologize for this test code, I forgot to mention that no test were ready in this version of the project, so maybe you think I haven't learned anything hehe is just I didn't had the time to do that part.